apps/src/reportAbuse.js

```diff
@@ -19,6 +19,8 @@ export const getChannelIdFromUrl = function(abuseUrl) {
   let match;
   if (abuseUrl.indexOf('codeprojects') >= 0 && abuseUrl.indexOf('weblab') < 0) {
     match = /.*codeprojects.*[^\/]+\/([^\/]+)/.exec(abuseUrl);
+  } else if (abuseUrl.indexOf('channelId=') >= 0) {
+    match = /\?channelId=(.*)/.exec(abuseUrl);
```

Comment:
Can you add a test in apps/test/unit/code-studio/components/ReportAbuseFormTest.js that tests this new logic?

Reply:
Ahh yes I should've done this initially-thank you. Done!